src/pages/workspaces/workspace/notebooks/AnalysisLauncher.js

```diff
@@ -292,7 +306,12 @@ const PreviewHeader = ({
                 Ajax().Metrics.captureEvent(Events.analysisLaunch,
                   { origin: 'analysisLauncher', source: tools.RStudio.label, application: tools.RStudio.label, workspaceName: name, namespace })
                 Nav.goToPath(appLauncherTabName, { namespace, name, application: 'RStudio' })
-              } else {
+                } else if (runtimeStatus === 'Stopped' && currentRuntimeTool === toolLabel) {
+               // we make it here because mode is undefined. we don't have modes for rstudio (that i'm aware of)
+                chooseMode('edit')
```

Comment:
why is `chooseMode` needed here? just to force a reload or sth?

Reply:
It's for line 284, so mode is defined and the Utils.cond(...) line will be false, and we can continue to checking logic on the other headers (see line 339 on)